### DISCUSSION TITLE
Supabase Projects can't be updated

### DISCUSSION THREAD

**Discussion by alQaisi** [supabase/supabase#8043] [Questions] [answered]
I can't update any of my project's settings, I got a confirmation that my project has been updated successfully, but when I refresh the page all of my changes disappear and the default settings return.
I've noticed an error with 403 code in the console.
anyone has the same issue here?

**Comment by GaryAustin1** [comment, marked as answer]
There were two reports of this on the discord channel last night also.
If you are still having trouble please contact support thru your help tab at top of dashboard.

Edit:
![image](https://user-images.githubusercontent.com/54564956/182649963-4e5436f0-a2c6-4974-948c-8c3fdde131aa.png)

**Reply by alQaisi** [reply]
It works fine now.
thanks for the help.
